### PR TITLE
GH-67 removed dead code and fixed test

### LINKED ISSUE #67: Eliminate dead code in RabbitMessageChannelBinder
The code in question is `RabbitMessageChannelBinder.createConsumerEndpoint(..)` (see below). Basically the bridge channel that is created in the first line is not used anywhere
```java
@Override
	protected MessageProducer createConsumerEndpoint(ConsumerDestination consumerDestination, String group,
													ExtendedConsumerProperties<RabbitConsumerProperties> properties) {
		DirectChannel convertingBridgeChannel = new DirectChannel();
		convertingBridgeChannel.setBeanFactory(this.getBeanFactory());

		. . .

		convertingBridgeChannel.setBeanName(baseQueueName + ".bridge");
. . .
```

### PR DESCRIPTION
**Pull request by olegz**
- removed dead code from RabbitMessageChannelBinder.createConsumerEndpoint(..)
- fixed RabbitBinderCleanerTests.testCleanStream()

Resolves #67

### CODE REVIEW

File: spring-cloud-stream-binder-rabbit/src/test/java/org/springframework/cloud/stream/binder/rabbit/RabbitBinderCleanerTests.java [hunk 1]
```diff
@@ -136,10 +136,19 @@ private void waitForConsumerStateNot(String queueName, int state) throws Interru
...
 					Map<String, Object> queueInfo = template.getForObject(uri, Map.class);
-					if (!queueInfo.get("consumers").equals(Integer.valueOf(state))) {
+					consumers = queueInfo.get("consumers");
+					if (consumers == null){
+						Thread.sleep(100);
+					}
+				}
+				assertThat(consumers != null);
+
+				while (n++ < 100) {
```
Comment (garyrussell) [inline spring-cloud-stream-binder-rabbit/src/test/java/org/springframework/cloud/stream/binder/rabbit/RabbitBinderCleanerTests.java]: We don't need this second loop - nothing changes inside of it.

I can remove during merge if you agree.

File: spring-cloud-stream-binder-rabbit/src/test/java/org/springframework/cloud/stream/binder/rabbit/RabbitBinderCleanerTests.java [hunk 2]
```diff
@@ -136,10 +136,19 @@ private void waitForConsumerStateNot(String queueName, int state) throws Interru
...
+					if (!consumers.equals(Integer.valueOf(state))) {
```
Comment (garyrussell) [inline spring-cloud-stream-binder-rabbit/src/test/java/org/springframework/cloud/stream/binder/rabbit/RabbitBinderCleanerTests.java]: We don't need this second loop - nothing changes inside of it.

This needs to go to 144 

    if (consumers == null || consumers.equals(Integer.valueOf(state))) {

I can fix during merge if you agree.

Comment (olegz) [inline spring-cloud-stream-binder-rabbit/src/test/java/org/springframework/cloud/stream/binder/rabbit/RabbitBinderCleanerTests.java]: That's true, go ahead fix during merge

**Comment by garyrussell** [comment]
Merged as c8cc4779e44214eb65641f92e95a2c218dae1eb9 with a bit of test polishing.

**Pull request closed without merging**